website/content/docs/install/index.mdx

```diff
@@ -1,9 +1,8 @@
 ---
 layout: docs
-page_title: Get Started
-description: |-
-  Installing Consul is simple. You can download a precompiled binary, compile
-  from source or run on Kubernetes. This page details these methods.
+page_title: Install Consul
+description: >-
+Install Consul to get started with service discovery and service mesh. Follow the installation instructions to download the precompiled binary, or use Go to compile from the source manually.
```

Comment:
```suggestion
Install Consul to get started with service discovery and service mesh. Follow the installation instructions to download the precompiled binary, or use Go to compile from source.
```

"compile from source" is more commonly used.

Reply:
When/if we rewrite this to also put the k8s instructions here we should update this to mention k8s.